lib/dradis/plugins/veracode/importer.rb

```diff
@@ -52,15 +52,21 @@ def parse_report_details(xml_detailedreport)
       end
 
       app_node.save
+      app_node
     end
 
-
     def parse_flaw(xml_flaw)
-      flaw_id = xml_flaw[:issueid]
-      logger.info{ "\t\t => Creating new issue (flaw issueid: #{ flaw_id })" }
+      cwe_id = xml_flaw[:cweid]
+      logger.info { "\t\t => Creating issue and evidence (flaw cweid: #{ cwe_id })" }
+
+      logger.info { "Adding report details (app_name: #{ app_name })" }
```

Comment:
You need an accessor for app_name to reference it here

Reply:
Removed